avm/res/resources/deployment-script/tests/e2e/private-network/main.test.bicep

```
@@ -0,0 +1,96 @@
+targetScope = 'subscription'
+
+metadata name = 'Using Private Endpoints'
```

```suggestion
metadata name = 'Using Private Networking'
```
It's not actually using the Private-Endpoint service (and I wish it would). 
Note, this updated requires a regeneration of the readme